libraries/AP_Landing/AP_Landing.cpp

```diff
@@ -142,7 +142,14 @@ const AP_Param::GroupInfo AP_Landing::var_info[] = {
     // @Group: DS_
     // @Path: AP_Landing_Deepstall.cpp
     AP_SUBGROUPINFO(deepstall, "DS_", 15, AP_Landing, AP_Landing_Deepstall),
-    
+
+    // @Param: _OPTIONS
```

Comment:
extra _ in the @Param, which is why CI is failing

Reply:
... so close.